src/Commands/DiffCommand.php

```diff
@@ -61,7 +61,7 @@ private function printJobDiff($jobName)
         $jobDiff = $jobComparisonBusinessCase->getJobDiff($jobName);
 
         foreach ($jobDiff as $property => $diff) {
-            $diffLines = array_reverse(explode(PHP_EOL, $diff));
+            $diffLines = explode(PHP_EOL, $diff);
```

Comment:
Is this supposed to have a visible effect? I don't see any difference in the output of "chapi diff" from this branch compared to what master does.